mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -192,6 +192,12 @@ public void updateUI(GeckoBundle controlCenterSettingsData) {
             disabledColors.add(ContextCompat.getColor(getContext(), category.categoryColorDisabled));
             blockedColors.add(ContextCompat.getColor(getContext(), category.categoryColorBlocked));
         }
+        if (entries.isEmpty()) {
+            entries.add(new PieEntry(1));
+            colors.add(ContextCompat.getColor(getContext(), R.color.cc_default_category_disabled));
```

Comment:
it's better to get the color ones into a variable and use it 3 times.

Reply:
But is it really better? Because the R file is a bunch of final static constants anyway.
@spacifici what's your opinion?